go.mod

```diff
@@ -76,8 +69,209 @@ require (
 	go.uber.org/zap v1.21.0
 	golang.org/x/crypto v0.0.0-20220131195533-30dcbda58838
 	golang.org/x/image v0.0.0-20210220032944-ac19c3e999fb
-	gopkg.in/go-playground/assert.v1 v1.2.1 // indirect
+	google.golang.org/protobuf v1.27.1
 	gopkg.in/go-playground/validator.v9 v9.31.0
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 	olympos.io/encoding/edn v0.0.0-20201019073823-d3554ca0b0a3
 )
+
+require (
```

Comment:
Why is there a second `require` section?

Reply:
Apparently:

>After a suitable module has been found, the go command will add a new [requirement](https://go.dev/ref/mod#go-mod-file-require) with the new module’s path and version to the main module’s go.mod file. This ensures that when the same package is loaded in the future, the same module will be used at the same version. __If the resolved package is not imported by a package in the main module, the new requirement will have an `// indirect` comment.__

https://go.dev/ref/mod#resolve-pkg-mod

Reply:
>The go command automatically adds `// indirect` comments for some requirements. An // indirect comment indicates that no package from the required module is directly imported by any package in the [main module](https://go.dev/ref/mod#glos-main-module).

https://go.dev/ref/mod#go-mod-file-require

Reply:
mmh, good point, I think something went funny during the rebase

Reply:
Seems to work fine, so whatever.